css/style.css

```
@@ -0,0 +1,333 @@
+/* fonts */
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../fonts/PTSans.woff2") format("woff2"),
+      url("../fonts/PTSans.woff") format("woff"),
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../fonts/PTSans-Bold.woff2") format("woff2"),
+      url("../fonts/PTSans-Bold.woff") format("woff"),
+  font-weight: 700;
+  font-style: normal;
+}
+/* Variables */
+:root{
+  --basic-black:#000000;
+  --basic-dark:#333333;
+  --basic-white:#ffffff;
+  --basic-white-focus:rgba(255, 255, 255, .3);
+  --basic-special: #666666;
+  --area-special:#F2F2F2;
+  --area-grey: #EEEEEE;
+  --area-grey-focus: #BDBBBC;
+   --area-grey-line: #E5E5E5;;
+  --area-blue:#81B3D2;
+  --area-blue-hover:#669EC0;
+  --area-blue-focus:#5496BD;
+  --area-brown: #766357;
+  --area-brown-hover: #604E43;
+  --area-brown-focus: #503E33;
+}
+
+/*Global*/
+.page{
+  background-color: var(--area-grey);
+}
+.page-body{
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0;
+  font-family: 'PT Sans' , Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: var(--basic-dark);
+  background-color: var(--basic-white);
+}
+a{
+  text-decoration: none;
+}
+img{
+  max-width: 100%;
+  height: auto;
+}
+ul{
```

не должно быть глобальных стилей https://up.htmlacademy.ru/htmlcss/30/criteries#d25
Вместо этого используй селектор по классу:
```suggestion
.list-reset{
```